### PR TITLE
Changed text to align with other apps

### PR DESCRIPTION
**Pull request by amobarak**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: steps/mini-petition/MiniPetition.content.json [hunk 1]
```diff
@@ -23,29 +23,39 @@
     "statementOfTruth": "Statement of Truth",
     "statementOfTruthDescription": "\"I believe that the facts stated in this application are true.\"",
     "coRespondent": "Co-respondent",
-
+    "reasonForDivorceSeparationTwoYearsBrokenDown": "The applicant and respondent have lived apart for a continuous period of at least two years immediately preceding the presentation of this application and the respondent consents to a decree being granted.",
```
Comment (hosainnet) [inline steps/mini-petition/MiniPetition.content.json]: I think we're missing some unit tests, as per https://github.com/hmcts/div-respondent-frontend/pull/214 for the conditional stuff

File: test/unit/steps/miniPetition.test.js [hunk 2]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
         'reasonForDivorceAdulteryCorrespondentNotNamed',
         'reasonForDivorceAdulteryWhere',
         'reasonForDivorceAdulteryWhen',
+        'reasonForDivorceSeparationTwoYearsBrokenDown',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceSeparationTwoYearsBrokenDown` doesn't seem to be used

File: test/unit/steps/miniPetition.test.js [hunk 3]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
...
+        'reasonForDivorceAdulteryDescription',
+        'reasonForDivorceInfo',
+        'reasonForDivorceStatement',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceStatement` Should be added to this test https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L807

File: test/unit/steps/miniPetition.test.js [hunk 4]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
         'reasonForDivorceAdulteryCorrespondentNotNamed',
         'reasonForDivorceAdulteryWhere',
         'reasonForDivorceAdulteryWhen',
+        'reasonForDivorceSeparationTwoYearsBrokenDown',
+        'reasonForDivorceAdulteryDescription',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceAdulteryDescription` Should be added to this test https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L807

File: test/unit/steps/miniPetition.test.js [hunk 5]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
...
+        'reasonForDivorceInfo',
+        'reasonForDivorceStatement',
+        'reasonForDivorceSeperationTwoYearsDecidedDate',
+        'reasonForDivorceSeperationTwoYearsLivingApartDate',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceSeperationTwoYearsDecidedDate` and `reasonForDivorceSeperationTwoYearsLivingApartDate` should be added to https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L858

File: test/unit/steps/miniPetition.test.js [hunk 6]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
...
+        'mostRecentDateIsSeperation',
+        'reasonForDivorceSeperationFiveYearsInfo',
+        'reasonForDivorceBehaviourBrokendDown',
+        'reasonForDivorceBrokendDown',
+        'reasonForDivorceDesertionInfo',
+        'reasonForDivorceDesertionDescription',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceDesertionInfo` and `reasonForDivorceDesertionDescription` should be added to 
https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L917

File: test/unit/steps/miniPetition.test.js [hunk 7]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
...
+        'reasonForDivorceSeparationTwoYearsBrokenDown',
+        'reasonForDivorceAdulteryDescription',
+        'reasonForDivorceInfo',
+        'reasonForDivorceStatement',
+        'reasonForDivorceSeperationTwoYearsDecidedDate',
+        'reasonForDivorceSeperationTwoYearsLivingApartDate',
+        'mostRecentDateIsSeperation',
+        'reasonForDivorceSeperationFiveYearsInfo',
+        'reasonForDivorceBehaviourBrokendDown',
+        'reasonForDivorceBrokendDown',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceBrokendDown` is not used

File: test/unit/steps/miniPetition.test.js [hunk 8]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
...
         'reasonForDivorceAdulteryWhen',
+        'reasonForDivorceSeparationTwoYearsBrokenDown',
+        'reasonForDivorceAdulteryDescription',
+        'reasonForDivorceInfo',
+        'reasonForDivorceStatement',
+        'reasonForDivorceSeperationTwoYearsDecidedDate',
+        'reasonForDivorceSeperationTwoYearsLivingApartDate',
+        'mostRecentDateIsSeperation',
+        'reasonForDivorceSeperationFiveYearsInfo',
+        'reasonForDivorceBehaviourBrokendDown',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceBehaviourBrokendDown` should be added to https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L841

File: test/unit/steps/miniPetition.test.js [hunk 9]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
         'reasonForDivorceAdulteryCorrespondentNotNamed',
         'reasonForDivorceAdulteryWhere',
         'reasonForDivorceAdulteryWhen',
+        'reasonForDivorceSeparationTwoYearsBrokenDown',
+        'reasonForDivorceAdulteryDescription',
+        'reasonForDivorceInfo',
+        'reasonForDivorceStatement',
+        'reasonForDivorceSeperationTwoYearsDecidedDate',
+        'reasonForDivorceSeperationTwoYearsLivingApartDate',
+        'mostRecentDateIsSeperation',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `mostRecentDateIsSeperation` here too https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L858

File: test/unit/steps/miniPetition.test.js [hunk 10]
```diff
@@ -460,6 +460,18 @@ describe(modulePath, () => {
...
+        'reasonForDivorceSeperationFiveYearsInfo',
```
Comment (hosainnet) [inline test/unit/steps/miniPetition.test.js]: `reasonForDivorceSeperationFiveYearsInfo` needs to be in https://github.com/hmcts/div-decree-nisi-frontend/blob/master/test/unit/steps/miniPetition.test.js#L874

**Comment by nsingamsetty** [comment]
This is tested and looks fine in the AOS and DN apps. But the same text should appear in the Mini Petition - Neil has raised the ticket for the i.e DIV-4328.Merging this

**Pull request merged**